src/test/java/com/dawn/jat/illuminati/IlluminatiApplicationTests.java

```diff
@@ -1,11 +1,22 @@
 package com.dawn.jat.illuminati;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.dawn.jat.illuminati.hello.HelloController;
 import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
 
+@RunWith(SpringRunner.class)
 @SpringBootTest
 class IlluminatiApplicationTests {
+    @Autowired
+    private HelloController controller;
+
     @Test
-    void contextLoads() {
+    void contextLoads() throws Exception {
+        assertThat(controller).isNotNull();
```

Comment:
We can call to IlluminatiApplication  main function in here, and test the main thread flow instead of this

Reply:
This logic sholud move to `src/test/java/com/dawn/jat/illuminati/hello/HelloControllerTest.java` as Unit Test
Furthermore, we can try calling controller.index() to see what the function will return

Reply:
Updated